### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by Alinahui** [NeymarL/captcha#1]
Hi! its written in my favorite Matlab, great work done but no documentation provided can you please help me to run this. please provide  read me or any other help file.

**Comment by NeymarL** [comment]
I'm sorry about that. This project primarily written in Matlab, to preprocess data and train with neural network. But later I find it didn't work as well as my expect, or say its prediction was very low. Then I decided to use [caffe](http://caffe.berkeleyvision.org/) model training those images with convolutional neural network. I'm not sure whether the Matlab code can work, because I didn't use it for a long time. So  you can just try to run the `train.m` script, but possibly it won't give you a nice predict result even thought it has no errors.

I can tell you each of these Matlab script's function.
- `readData.m` : read image and transform to unroll them
- `removeBorder.m` : remove the border of a captcha image
- `splitCaptcha.m` : split a captcha into n single letter or number
- `sigmoid.m` : the sigmoid function
- `nnCostFunction.m` : implement the neural network cost function
- `fmincg.m` : advanced gradient descent algorithm
- `makeXY.m` : trans raw data into features(X) and lable(Y)
- `predict.m` : Predict the label of an input given a trained neural network
- `train.m` : train data

By the way, I run these script in [GNU Octave](https://www.gnu.org/software/octave/) instead of Matlab, so it maybe have compatibility problem.

**Comment by Alinahui** [comment]
NeymarL thanks for your response.
